e2e/cypress/integration/menus/menu_actions_spec.js

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Since the test belongs to "Team Settings" functional area, please move spec file to `team_settings`.